package.json

```diff
@@ -24,13 +24,14 @@
   "dependencies": {
     "css-vars-ponyfill": "^1.8.0",
     "firefox-iframe-getcomputedstyle": "^1.0.0",
-    "quasar-framework": "^0.15.10"
+    "quasar-framework": "^0.17.14"
   },
   "bedrock": {
     "browserDependencies": "all",
     "manifest": {
       "quasar-framework": {
         "main": "dist/umd/quasar.mat.umd.js",
+        "module": "dist/umd/quasar.mat.umd.js",
```

Comment:
Is adding "module" here the right thing to do? We didn't need it before ... what changed?

Reply:
Not doing so makes bedrock-views pull in the esm version of quasar causing it to break during development mode. What changed was the addition of "module" in quasar's package.json https://github.com/quasarframework/quasar/commit/0ac2e638f5ac47d522aaa37205fd81db86c6b6b1

Reply:
I see. Good to know. Did you check to see what happens when we optimize now? Does it include multiple versions of the same lib?

Reply:
It does not.